code/modules/bitrunning/objects/byteforge.dm

```diff
@@ -63,5 +63,5 @@
 /obj/machinery/byteforge/proc/start_to_spawn(obj/cache)
 	flicker()
 
-	addtimer(CALLBACK(src, PROC_REF(spawn_cache), cache), 1 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE|TIMER_STOPPABLE)
+	addtimer(CALLBACK(src, PROC_REF(spawn_cache), cache), 1 SECONDS, TIMER_STOPPABLE)
```

Comment:
Without a reference to its id, a stoppable timer cannot be stopped anyway. Is the flag even necessary now?